tools/10x_bamtofastq/10x_bamtofastq.xml

```diff
@@ -0,0 +1,74 @@
+<tool id="10x_bamtofastq" name="Convert a 10X BAM file to FASTQ" version="1.4.1">
+    <requirements>
+        <requirement type="package" version="1.4.1">10x_bamtofastq</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        bamtofastq --nthreads "\${GALAXY_SLOTS:-4}" --reads-per-fastq 10000000000
```

Comment:
With such a high value we try to generate only a single FASTQ file per matepair. Decreasing this value generates multiple FASTQ files. We don't loose any reads because we are moving all the FASTQs to the output collection.

Reply:
ok.